### ISSUE TITLE
Keep the apostrophe/quotation marks in pyproject version replacement

### ISSUE THREAD

**Issue by crash5** [mtkennerly/poetry-dynamic-versioning#156]
I'm using apostrophe `'` for the config in my pyproject file like `version = '0.0.0'` but on poetry install/build and poetry dynamic-versioning commands the plugin replaces it with quotation `"` marks.

**Comment by mtkennerly** [comment]
This behavior would be coming from tomlkit. I do see this issue that says it was fixed: https://github.com/sdispater/tomlkit/issues/112 , although it doesn't say in which version. I just tested with the latest tomlkit version, and I was able to reproduce it, so I've opened a new ticket with them: https://github.com/sdispater/tomlkit/issues/325 . This affects the `poetry version` command as well, even without the plugin involved.

The plugin defers to whichever tomlkit version Poetry is using, so if it's fixed in tomlkit and then included in a Poetry update, then it'll be fixed for the plugin as well. Beyond that, it's out of my control, unfortunately.

In the meantime, I would suggest using double quotes just to avoid the issue.
